lib/validator/sfValidatorDate.class.php

```diff
@@ -117,6 +117,7 @@ protected function doClean($value)
       else
       {
         $dateMax  = new DateTime($max);
+        $dateMax->setTimezone(new DateTimeZone(date_default_timezone_get()));
```

Comment:
Hello @nederdirk,

Here is not an issue with the time zone but the way where the date time are compared.

The proper way to do it is to compare timestamps.

As the timestamp does not depends on the timezone then all issue will be resolved there.